### DISCUSSION TITLE
New Varactor Library for ngspice

### DISCUSSION THREAD

**Discussion by tomhajjar** [ra3xdh/qucs_s#758] [General]
Attached is a new library for ngspice called Varactor_RF.lib. Also attached is a project used to test the models and develop the library, Models include packaging parasitics. Should be used with the latest ngspice build that allows larger Vj and M.

![2024-06-15_185048](https://github.com/ra3xdh/qucs_s/assets/102838604/ae9d8133-1742-433a-9fcd-afccaf92b315)

![2024-06-15_185516](https://github.com/ra3xdh/qucs_s/assets/102838604/ff1bfa6d-1cfc-4fb3-bcf1-e1cbf60a652c)

![2024-06-15_214751](https://github.com/ra3xdh/qucs_s/assets/102838604/f1b1bc6e-6d63-43ea-80a8-8da1ff202285)

[Varactor_ngspice_prj.zip](https://github.com/user-attachments/files/15860461/Varactor_ngspice_prj.zip)

**Comment by ra3xdh** [comment]
Thanks! I will upload a new library soon. See #759.

**Comment by ra3xdh** [comment]
The new library has been uploaded to the repository in #759.
